endurance-page-cache.php

```diff
@@ -238,6 +238,15 @@ public function to_snake_case( $value, $delimiter = '_' ) {
 			return $value;
 		}
 
+		/**
+		 * Checks if this environment caches requests on the current filesystem
+		 *
+		 * @return bool True if uses file system to cache
+		 */
+		public function use_file_cache() {
+			return false === strpos( dirname( __FILE__ ), 'public_html' );
+		}
```

Comment:
One possible little improvement here is instead of a method, we could should probably just make it an object property set in `__construct()`.

Reply:
@earnjam Don't all our environments have `public_html` in the path? I'm not following the logic on how this determines when the file cache is used...

Reply:
It doesn't appear so. All I did was move this logic that exists in multiple places into a function. 

In the current version of the plugin, this check is used to determine whether or not to:
- [store static cached files on the filesystem](https://github.com/bluehost/endurance-page-cache/blob/3b4cbadd4de4e40d62a0bfceadccad1c21f03f50/endurance-page-cache.php#L494)
- [add additional rewrite rules to .htaccess for handling static file-based page cache](https://github.com/bluehost/endurance-page-cache/blob/3b4cbadd4de4e40d62a0bfceadccad1c21f03f50/endurance-page-cache.php#L833-L845)
- [toggle the nginx page cache setting when the cache level is changed](https://github.com/bluehost/endurance-page-cache/blob/3b4cbadd4de4e40d62a0bfceadccad1c21f03f50/endurance-page-cache.php#L1151)

Maybe @MikeHansenMe can chime in here to confirm.